packages/developer-portal/src/core/connect-session.ts

```diff
@@ -0,0 +1,12 @@
+import { ReapitConnectBrowserSession } from '@reapit/connect-session'
+// Referencing config direct rather than via window because window.reapit.config not available at compile time
+// and throws a runtime error
+import config from '../../config.json'
```

Comment:
So this is not needed now, you can just reference window.reapit.config - you will need to lazy load the app component in the index.tsx see below comment for how to do this